src/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Project Le News</title>
+    <meta charset="UTF-8" />
```

Comment:
How about a favicon? :)
What about a description?

Reply:
Is a small description enough or should I add some more infos?